howso/get_cases.amlg

```diff
@@ -354,7 +352,7 @@
 
 	;return the query conditions for cases that match the specified condition
 	;parameters:
-	; condition_session: optional, if specified ignores condition and returns cases for the specified session id
+	; condition_session: 
```

Comment:
I figure this was not removed on purpose

Reply:
Not intentionally for sure, restored and addressed in: [d54a657](https://github.com/howsoai/howso-engine/pull/239/commits/d54a6575c30a5c75415b2cc52e96d51cfb0406c6)